modules/label/parser.go

```diff
@@ -0,0 +1,126 @@
+// Copyright 2023 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package label
+
+import (
+	"errors"
+	"fmt"
+	"strings"
+
+	"code.gitea.io/gitea/modules/options"
+
+	"gopkg.in/yaml.v3"
+)
+
+type labelFile struct {
+	Labels []*Label `yaml:"labels"`
+}
+
+// ErrTemplateLoad represents a "ErrTemplateLoad" kind of error.
+type ErrTemplateLoad struct {
+	TemplateFile  string
+	OriginalError error
+}
+
+// IsErrTemplateLoad checks if an error is a ErrTemplateLoad.
+func IsErrTemplateLoad(err error) bool {
+	_, ok := err.(ErrTemplateLoad)
+	return ok
+}
+
+func (err ErrTemplateLoad) Error() string {
+	return fmt.Sprintf("Failed to load label template file '%s': %v", err.TemplateFile, err.OriginalError)
```

Comment:
```suggestion
	return fmt.Sprintf("Failed to load label template file '%s': %w", err.TemplateFile, err.OriginalError)
```

Reply:
There is is no point in this as this returns string and also imho %w is only supported in `fmt.Errorf`

Reply:
Yep, I confused it with `fmt.Errorf` as it looked like you would return an error.

Reply:
`fmt.Sprintf does not support error-wrapping directive %w`